recipes/evalica/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "evalica" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/evalica-{{ version }}.tar.gz
+  sha256: 49ce78da9dea0b75ece86ba0ba2159ee3550a2c604b73675db82a064e1f68164
+
+build:
+  entry_points:
+    - evalica = evalica.__main__:main
+  script:
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ stdlib("c") }}
+    - {{ compiler('rust') }}
+    - python  # [build_platform != target_platform]
```

Comment:
If you add this, you also need to add cross-python. Better let the arch migrator do it for you.
```suggestion
```